app/controllers/experiments_controller.rb

```diff
@@ -675,9 +675,10 @@ def extract_types_for_parameters(simulation_runs)
   def extract_types_for_moes(simulation_runs)
     array_for_moes_types = []
 
-    first_run = simulation_runs.where('$and' => [{result: {'$exists' => true}}, {result: {'$ne' => nil}}]).first
+    first_run = simulation_runs.where('$and' => [{result: {'$exists' => true}}, {result: {'$ne' => {}}}, {result: {'$ne' => nil}} ]).first
```

Comment:
@dkrol - może warto zamienić to query na użycie: https://docs.mongodb.org/v3.0/reference/operator/query/nin/ ?

``the field value is not in the specified array or the field does not exist.``